docs/source/dev-guide/plugins/environment_specifiers.rst

```diff
@@ -0,0 +1,71 @@
+======================
+Environment Specifiers
+======================
+
+Conda can create environments from several file formats. Currently, conda natively
+supports creating environments from:
+
+* a yaml `environment.yaml`
+* an "explicit" text file
```

Comment:
```suggestion
* A :ref:`YAML environment.yaml file <create-env-file-manually>`
* An :ref:`"explicit" text file <identical-conda-envs>`
```

We could make this slightly better with direct links to other parts of the documentation 😄.

Reply:
I also tested this myself, and verified that's the correct format too.